app/views/event/register/_email_check.html.haml

```diff
@@ -0,0 +1,13 @@
+-#  copyright (c) 2024, schweizer alpen-club. this file is part of
+-#  hitobito_sac_cas and licensed under the affero general public license version 3
+-#  or later. see the copying file at the top-level directory or at
+-#  https://github.com/hitobito/hitobito_sac_cas.
+
```

Comment:
Hier fehlt im Vergleich zum Youth nur das <p> oder? Ist es realistisch dort den Text in den locales einfach blank `""` zu lassen?